great-idea-website/css/index.css

```diff
@@ -63,4 +63,169 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+h1 {
+	font-size: 80px;
+	text-align: center;
+}
+
+h2 {
+	font-size: 20px;
+}
+
+hr {
+	width: 70%;
+	margin-left: 150px;
+	margin-right: auto;
+}
+
+body {
+	margin: 0px 50px 0px 50px;
+}
+
+.nav-bar {
+	display: inline-block;
+	width: 100%;
+}
+
+nav {
+	display: inline-block;
+	margin-left: 75px;
+}
+
+.nav-box-one {
+	display: inline-block;
+	
+}
+
+.nav-box-two {
+	display: inline-block;
+	margin-top: 15px;
+	margin-left: 125px;
+}
+
+a {
+	font-size: 18px;
+	color: gray;
+	text-decoration: none;
+	margin-left: 50px;
+}
+
+/* Box for top section */
+.box-section-one {
+	display: inline-block;
+	width: 100%;
+	height: 450px;
+	margin-top: 15px;
+	margin-bottom: 75px;
+}
+
+/* Box for Slogan & Start button */
+.box-content-one {
+	display: inline-block;
+	height: 200px;
+	width: 300px;
+	margin-left: 170px;
+}
+
+/* Innovation on demand Box */
+.box-one {
+	display: inline-block;
+	margin: 0px 0px 10px 10px;
+}
+
+/* Get Started Button */
+.box-two {
+	display: inline-block;
+	font-size: 20px;
+	color: gray;
+	width: 175px;
+	height: 30px;
+	text-align: center;
+	margin-left: 80px;
+	padding-top: 2px;
+	border: 2px solid gray;
+}
+
+/* Box Right with Image */
+.box-content-two {
+	display: inline-block;
+	margin: 60px 0px 0px 150px;
+}
+
+/* Middle Section Above Banner Image in Middle */
+
+/* Section 2 */
+.box-section-two {
```

Comment:
Instead of the complicated/non responsive margins here, to get closer to center, I would have set a percentage width of each box section, so for the first two they would be 50%, then center those divs in each section so the margins are more uniform.

Reply:
This would also work for box section one.